.mergify.yml

```diff
@@ -393,6 +380,14 @@ pull_request_rules:
         labels:
           - "backport"
         title: "[{{ destination_branch }}](backport #{{ number }}) {{ title }}"
+  - name: backport patches to 8.19 branch
+    conditions:
+      - merged
+      - label=backport-8.19
+    actions:
+      backport:
```

Comment:
See https://github.com/elastic/elastic-agent/blob/8c4c195d8bc925a9938dd881104ad1cd7e7915f1/.mergify.yml#L13-L20

No need to add the default values anymore.

Likely, in a follow-up we will be remove all those duplicated values in all the backports